affinity/cpp-20/d0796r3.md

```diff
@@ -539,8 +593,6 @@ The `execution_resource` class provides an abstraction over a system's hardware,
 
 *Requires:* `const_iterator` to model `RandomAccessIterator` with the value type `execution_resource::value_type`.
```

Comment:
Does "with the value type X" mean "`is_same_v<iterator_traits<const_iterator>::value_type, X>` is `true`"?

Reply:
Also, I think the preferred way to say this is "`const_iterator` satisfies the *Cpp17RandomAccessIterator* requirements."

Reply:
Yes, that's right, I've made the wording more explicit here, and also took your suggestion for the random access iterator requirements.